src/d3a/d3a_core/sim_results/export_unmatched_loads.py

```diff
@@ -167,36 +156,29 @@ def _get_hover_info(cls, indict, slot_time):
         ul_count = 0
         for child_name, child_ul_list in indict.items():
             for time in child_ul_list:
-                if time.hour == slot_time.hour and \
-                   time.day == slot_time.day and \
-                   time.month == slot_time.month and \
-                   time.year == slot_time.year:
+                if int(time[11:13]) == int(slot_time[11:13]) and \
+                   int(time[8:10]) == int(slot_time[8:10]) and \
+                   int(time[5:7]) == int(slot_time[5:7]) and \
+                   int(time[:4]) == int(slot_time[:4]):
```

Comment:
I think you could parse a pendulum object to `update_unmatched_loads` and then you can save this here.

Reply:
Yes, I could parse it to pendulum object but again I have to re-convert to string to save it in results string, so I would actually prefer to save some back and forth datetime conversion

Reply:
I dont see your point the pendulum object for the current_time_slot is already there, you only have to parse it to `update_unmatched_loads`. There is only the `.format(DATE_TIME_FORMAT)` that has to be done here. But in your implementation, you need to transform 8 strings into integer which is not very effective either. And it is not very elegant IMO, too. Sorry to say.

Reply:
Thanks for the offline explanation, you are right, in the future the results service will not get the pendulum object but a string from the d3a. Does make any sense to convert it to pendulum object. before just to have pretty code here.
However, i wonder if you could get rid of the `int()` as you comparing characters which each other and don't use `>` or `<`

Reply:
`However, i wonder if you could get rid of the int() as you comparing characters which each other and don't use > or <`

Yup, just checked it, no need for an explicit conversion to `int` here, `str` conversion also works fine here